### ISSUE TITLE
Ammend Typos and Expand Mock-Data

### ISSUE THREAD

**Issue by salvolpe** [COVID-19-electronic-health-system/Coronalytics#15]
I'm beginning to preprocess the data and I've noticed that there are a couple of one-letter typos in the labels:
- bluLipsFace --> blueLipsFace
- breathShortnes --> breathShortness
- gastrointestinnal --> gastrointestinal
- cadiosvascular --> cardiovascular
- comorbidites --> comorbidities

Also, I wanted to expand the data to include:
- Demographics, based on the options provided currently in [Health](https://docs.google.com/spreadsheets/d/1tjeTH99QZSmyfG-TvX5seSYKa03yAHdMWrZt1z9521I/edit?usp=drive_web&ouid=104521265234292544579)
- - Specifically: race/ethnicity, age, gender, postal code
- Additional samples (maybe get up to at least 100 unique users, if not more)
- Eventually with more information on the effect of comorbidities, we can add illnesses to that category. But at the moment, I don't think we'll have any access to that information

**Comment by ngiangre** [comment]
Do we need to update the mock data based on the translations update @pavel-ilin

**Comment by pavel-ilin** [comment]
> Do we need to update the mock data based on the translations update @pavel-ilin?

Yes we do need to modify mock data. I can start working on that on Monday.

**Comment by ngiangre** [comment]
@salvolpe can you update on how usable the previous mock data was for your analysis? And recommend any useful modifications? Also update what analysis you did :)

**Comment by salvolpe** [comment]
@ngiangre @pavel-ilin The mock-data really wasn't useful, unfortunately, because of the limited relationships between the features and near-pure randomness of the construction. I'd love to set up a time to chat this weekend on ways that we make the mock data more robust for a model (that hopefully) works

**Comment by pavel-ilin** [comment]
I believe that issue is not ready to be closed. In pr #20 only show the structure of the mock data.

Next step will be to add logic of how we generate data and what parameter increase probability of having a covid.
